docs/examples/requirement-network.cwl

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.2
```

Comment:
Not sure if relevant, but the `NetworkAccess` feature seems to be available from v1.1.
https://github.com/common-workflow-language/cwl-v1.1/releases#:~:text=Added%20NetworkAccess%20feature%2C%20allowing%20to%20indicate%20whether%20a%20process%20requires%20outgoing%20network%20access.

Reply:
Yes. This shouldn't be a problem as long as the specified version is minimally `v1.1` to support it. It is still available in later versions.